src/python/zquantum/core/interfaces/optimizer_test.py

```diff
@@ -35,11 +36,10 @@ def test_optimization(self):
             self.assertIn("opt_params", results.keys())
             self.assertIn("history", results.keys())
 
-    def test_optimization_simple_function(self):
+    def test_optimizer_succeeds_with_optimizing_simple_function(self):
```

Comment:
I would rename "simple_function" to "sum_of_squares".

Reply:
Done!